rpm/macros.d/macros.go-rpm

```diff
@@ -8,6 +8,39 @@
 # The macros necessary at %%setup and srpm stage are in the sister file
 # macros.go-srpm
 
+# Default filtering policy
+# Complete or replace the following variables in your spec file if this policy
+# does not apply to your project
+
+# Flags to filter examples
+# The average Go project example code is incomplete and obsolete. It will
+# fail to compile as-is, or even panic. It will poison the dependency
+# generator with bogus or legacy import path requirements. Even when it it
+# clean and up to date, it will require many third party components not needed
+# in production. Therefore:
+# – never install Go project example code in %%{gopath}. Publish is as %%doc.
+# — ask politely projects that publish example code in some other directory
+#   than _examples to rename it.
+# – never rely on third party examples being available or working in another
+#   project. This is *not* production code. Ask upstreams politely to remove
+#   tests or code that import other project examples.
+%goignoreflags_examples -r '.*example.*'
```

Comment:
This is too general, one could define `flexamplegrin` package for example.

Reply:
One could do a lot of things and with coulds we would never do anything Do you have an actual real-life collision example?

Reply:
Let's mention that in the guidelines and state that if the default example exclusion policy needs to change, the `goignoreflags_examples` macro needs to be set to a different value or to `''`.